api_v3/services/LiveStreamService.php

```diff
@@ -414,16 +415,18 @@ public function isLiveAction ($id, $protocol = null)
 					$config = $liveStreamEntry->getLiveStreamConfigurationByProtocol($hlsProtocol, requestUtils::getProtocol());
 					if ($config){
 						$url = $config->getUrl();
+						$backupUrl = $config->getBackupUrl();
 						$protocol = $hlsProtocol;
 						$dpda->setFormat($protocol);
 						break;
 					}
 				}
 
-				KalturaLog::info('Determining status of live stream URL [' .$url. ']');
+				KalturaLog::info("Determining status of live stream URL [ $url ] and Backup URL [ $backupUrl ]");
 				$urlManager = DeliveryProfilePeer::getLiveDeliveryProfileByHostName(parse_url($url, PHP_URL_HOST), $dpda);
-				if($urlManager)
-					return $this->responseHandlingIsLive($urlManager->isLive($url));
+				$urlManagerBackup = DeliveryProfilePeer::getLiveDeliveryProfileByHostName(parse_url($backupUrl, PHP_URL_HOST), $dpda);
+				if ($urlManager || $urlManagerBackup)
+					return $this->responseHandlingIsLive(self::isLiveByUrlManager($urlManager, $url) || self::isLiveByUrlManager($urlManagerBackup, $backupUrl));
```

Comment:
isLiveByUrlManager cannot accept null, so in case no backup - you will have an issue.
One option is to remove the type in the function arguments.

Reply:
Done

Reply:
Done